Events/PinwheelInputAmountEvent.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  PinwheelInputAmountEvent.swift
+//  PinwheelSDK
+//
+//  Created by Robby Abaya on 2/12/21.
+//
+
+import Foundation
+
+public struct PinwheelInputAmountEvent: Codable {
```

Comment:
Type and eventName are common to all events so this type of data structure seems redundant since I think you can just have different data types for the payloads. Not sure if this gains something I'm missing though.

Reply:
Inheritance with the Codable protocol isn't straightforward, and I thought this was a simpler way to go. What do you think about keeping this for now, and discussing this further for a possible minor version update.

Reply:
Yeah definitely